web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -83,7 +95,7 @@ h3, h4,
 
 #site-header {
     background: none;
-    border-image-source: url(images/tile-y-border.jpg);
+    border-image-source: url(images/tile-y.jpg);
```

Comment:
I couldn’t wrap my head around how to make double dpi images work inside of `border-image`, and when I tried it out, I realised, in a 4px high border, you really can’t notice when an image has been scaled up anyway. Do you disagree?